bigquery/google/cloud/bigquery/magics.py

```diff
@@ -433,10 +431,16 @@ def _make_bqstorage_client(use_bqstorage_api, credentials):
     if not use_bqstorage_api:
         return None
 
-    if bigquery_storage_v1beta1 is None:
-        raise ImportError(
+    try:
+        from google.cloud import bigquery_storage_v1beta1
+    except ImportError as err:
+        customized_error = ImportError(
             "Install the google-cloud-bigquery-storage and fastavro packages "
             "to use the BigQuery Storage API."
         )
+        six.raise_from(customized_error, err)
 
-    return bigquery_storage_v1beta1.BigQueryStorageClient(credentials=credentials)
+    return bigquery_storage_v1beta1.BigQueryStorageClient(
+        credentials=credentials,
+        client_info=client_info.ClientInfo(user_agent=IPYTHON_USER_AGENT),
```

Comment:
I think we need the other ClientInfo, since this is a grpc client. [`google.api_core.gapic_v1.client_info.ClientInfo`](https://googleapis.github.io/google-cloud-python/latest/core/client_info.html#google.api_core.gapic_v1.client_info.ClientInfo)

That import can fail if grpc libraries aren't installed, so the import should be in the try/except block.